<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TimeEditWidget</class>
 <widget class="QWidget" name="TimeEditWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>360</width>
    <height>280</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>B Nazanin</family>
    <pointsize>-1</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">color:black;
font-style:bold;
font-size:17px;
font-family:&quot;B Nazanin&quot;;</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>1</number>
   </property>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="label_4">
     <property name="font">
      <font>
       <family>B Nazanin</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="styleSheet">
      <string notr="true">color:white;
font-style:bold;
font-size:17px;
font-family:&quot;B Nazanin&quot;;</string>
     </property>
     <property name="locale">
      <locale language="Persian" country="Iran"/>
     </property>
     <property name="text">
      <string>)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="secondsSpinBox">
     <property name="font">
      <font>
       <family>B Nazanin</family>
       <pointsize>-1</pointsize>
      </font>
     </property>
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">color:white;
font-style:bold;
font-size:17px;
font-family:&quot;B Nazanin&quot;;</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="minutesSpinBox">
     <property name="maximum">
      <number>59</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_2">
     <property name="styleSheet">
      <string notr="true">color:white;
font-style:bold;
font-size:17px;
font-family:&quot;B Nazanin&quot;;</string>
     </property>
     <property name="text">
      <string>:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="hoursSpinBox">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimum">
      <number>0</number>
     </property>
     <property name="maximum">
      <number>999</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label_3">
     <property name="styleSheet">
      <string notr="true">color:white;
font-style:bold;
font-size:17px;
font-family:&quot;B Nazanin&quot;;</string>
     </property>
     <property name="text">
      <string>(</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
